core/services/log/helpers_test.go

```diff
@@ -185,6 +197,8 @@ func (listener simpleLogListener) HandleLog(lb log.Broadcast) {
 
 	if !consumed {
 		listener.received.uniqueLogs = append(listener.received.uniqueLogs, lb.RawLog())
+	} else {
+		logger.Warnf("Listener %v: Log was already consumed!", listener.name)
```

Comment:
How can this actually happen?

Reply:
Normally in case of some heavy back-and-forth reorgs. But this warn is just for tests, i needed it briefly, don't remember for what exactly